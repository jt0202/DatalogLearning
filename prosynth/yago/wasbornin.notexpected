guil_falcon	port_hueneme_california
r_h_tawney	guangzhou
walter_vickery	new_york_city
susan_kiefel	new_york_city
frederick_barton_maurice	rome
