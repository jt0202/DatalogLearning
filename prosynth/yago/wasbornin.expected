siva_selliah	colombo
paul_capellani	paris
fumiko_enchi	tokyo
sergey_yustinovich_bagotsky	moscow
mihailo_janković	belgrade
prince_francis_xavier_of_saxony	dresden
prince_michael_feodorovich_of_russia	paris
pep_leadley	hamilton_ontario
prince_axel_of_denmark	copenhagen
hedwig_von_trapp	zell_am_see
john_george_i	lt_Elector_of_SaxonyDresdengt
florence_harding	marion_ohio
feodor_i_of_russia	moscow
frederick_christian	lt_Elector_of_SaxonyDresdengt
daniel_rodney	lewes_delaware
eugèneanatole_demarçay	paris
albert_g_schmedeman	madison_wisconsin
rizwanur_rahman	kolkata
albert_tocco	chicago
bill_hopkin	newport
grand_duke_alexander_alexandrovich_of_russia	saint_petersburg
arthur_gould_rugby_player	newport
princess_maria_ludwiga_theresia_of_bavaria	lindau
rose_of_viterbo	viterbo
eugen_filotti	bucharest
joseph_warder	jeffersonville_indiana
adrienmarie_legendre	paris
witold_leszczyński	ltŁódźgt
mantarō_kubota	tokyo
abel_gance	paris
felix_of_nola	nola
clifford_walker	monroe_georgia
albert_v	lt_Duke_of_BavariaMunichgt
liverus_hull	westfield_massachusetts
alice_cogswell	hartford_connecticut
vere_ponsonby	lt_9th_Earl_of_BessboroughLondongt
jean_bernard_physician	paris
muhammad_ali_jinnah	karachi
roy_mcelroy	auckland
tzeni_karezi	athens
elizabeth_siddal	london
sybil_arundale	london
lucien	lt_Lord_of_MonacoMonacogt
reg_plummer_rugby_player	newport
isabel_burton	london
guido_gozzano	turin
walter_bestmann	lübeck
joseph_babinski	paris
joão_do_canto_e_castro	lisbon
